automation/UNIX/test-lnt.sh

```diff
@@ -0,0 +1,36 @@
+# Test clang on UNIX using Visual Studio Team Services
+
+set -ue
+set -o pipefail
+set -x
+
+if [ -z "$LNT" ]; then
+  exit 0;
+fi
+
+CLANG=${LLVM_OBJ_DIR}/bin/clang
+RESULT_LOG="${LNT_RESULTS_DIR}/result.log"
+
+
+if [ ! -e "$CLANG" ]; then
+  echo "clang compiler not found at $CLANG"
+  exit 1
+fi
+
+"$LNT_SCRIPT" runtest nt --sandbox "$LNT_RESULTS_DIR" --no-timestamp \
```

Comment:
Thanks - the script deletes the results directory each time it runs, so there will be interaction with prior runs.  

I didn't want to use a timestamped directory name because it means that we the script does not know the precise directory name.  That in turn would make it harder find files later to upload for retention.  IN VSTS, agents cache files, but are not used for long-term retention of files.